src/srv.rs

```diff
@@ -189,11 +191,28 @@ impl SrvResolver {
                 "replicaset" => {
                     config.replica_set = Some(parts[1].into());
                 }
+                "loadBalanced" => {
```

Comment:
This needs to be lowercase. See Line 187.

Reply:
Good catch, thank you!